### PR TITLE
10175 add module crm_batch_leads

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: crm_batch_leads/README.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Elico Batch Leads & Email Templates
```
Comment (duanyp1991) [inline crm_batch_leads/README.rst]: remove Elico.

File: crm_batch_leads/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Elico CRM & Projects ACL',
```
Comment (duanyp1991) [inline crm_batch_leads/__openerp__.py]: remove elico

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: crm_batch_leads/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
```
Comment (siyuancn) [inline crm_batch_leads/__init__.py]: Please use new template for the copyright

File: crm_batch_leads/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
```
Comment (siyuancn) [inline crm_batch_leads/__openerp__.py]: Please use new template for the copyright

File: crm_batch_leads/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'CRM & Projects ACL',
+    'version': '1.1',
```
Comment (siyuancn) [inline crm_batch_leads/__openerp__.py]: wrong version

File: crm_batch_leads/wizard/batch_lead.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (siyuancn) [inline crm_batch_leads/wizard/batch_lead.py]: Please use new template for the copyright

**Comment by siyuancn** [comment]
Please user the structure:
A module is organised in a few directory:

```
controllers/: contains controllers (http routes)
data/: data xml
demo/: demo xml
models/: models definition
report/: reporting models (BI/analysis), Webkit/RML print report templates
static/: contains the web assets, separated into css/, js/, img/, lib/, ...
views/: contains the views and templates, and QWeb report print templates
wizards/: wizard model and views
```

and License.

**Comment by elicoidal** [comment]
@siyuancn Not mandatory for the history upload.
We will do it for any new module

File: crm_batch_leads/README.rst [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+   :alt: License: AGPL-3
+
+===================================
+Batch Leads & Email Templates
+===================================
+
+Security Rules for deleting CRM Leads/Opportunities and Projects Tasks
+----------------------------------------------------------------------
+* 1.1 Add menu for email template in sales
+* 1.1 Add batch for leads
```
Comment (elicoidal) [inline crm_batch_leads/README.rst]: remove numbers

File: crm_batch_leads/README.rst [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+Credits
+=======
+
+
+Contributors
+------------
+
+* Bob Lucas <luo.yumeng@elico-corp.com>
+
+Maintainer
```
Comment (elicoidal) [inline crm_batch_leads/README.rst:36]: Elico Maintainer

File: crm_batch_leads/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    'category': 'Generic Modules/Others',
+    'complexity': 'easy',
```
Comment (elicoidal) [inline crm_batch_leads/__openerp__.py:8]: remove this key

File: crm_batch_leads/__openerp__.py [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    'author': 'Elico Corp,Odoo Community Association (OCA)',
```
Comment (elicoidal) [inline crm_batch_leads/__openerp__.py]: remove OCA

Comment (Rona111) [inline crm_batch_leads/__openerp__.py]: @elicoidal  should i remove all the author "OCA " before the module push to OCA.

Comment (elicoidal) [inline crm_batch_leads/__openerp__.py]: Yes @rona111: sorry about that.
When we push we will adapt the README again :(

Comment (Rona111) [inline crm_batch_leads/__openerp__.py]: @elicoidal  so ,when i push to elido odoo,I need keep the author with OCA . and when we push we will adapt the README again?

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: crm_batch_leads/__openerp__.py (hunk 10 above)
Comment (elicoidal) [inline crm_batch_leads/__openerp__.py]: For Elico no need OCA.
If the module is pushed later to OCA then we add OCA as author

Comment (Rona111) [inline crm_batch_leads/__openerp__.py]: @elicoidal ok,I will remove OCA.

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by siyuancn** [comment]
LGTM

File: crm_batch_leads/README.rst [hunk 11]
```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Batch Leads & Email Templates
+=============================
+
+Security Rules for deleting CRM Leads/Opportunities and Projects Tasks
```
Comment (elicoidal) [inline crm_batch_leads/README.rst]: Remove this 2 lines

File: crm_batch_leads/__openerp__.py [hunk 12]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'CRM & Projects ACL',
+    'version': '8.0.1.0.2',
+    'category': 'Generic Modules/Others',
+    'complexity': 'easy',
+    'author': 'Elico Corp',
+    'website': 'http://www.openerp.net.cn/',
```
Comment (elicoidal) [inline crm_batch_leads/__openerp__.py]: www.elico-corp.com

File: crm_batch_leads/email_template.xml [hunk 13]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
```
Comment (elicoidal) [inline crm_batch_leads/email_template.xml]: remove this feature

File: crm_batch_leads/README.rst [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+----------------------------------------------------------------------
+
+* Add menu for email template in sales
+* Add batch for leads
```
Comment (elicoidal) [inline crm_batch_leads/README.rst]: Adds wizard to batch process the CRM opportunities.

File: crm_batch_leads/README.rst [hunk 15]
```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Batch Leads & Email Templates
```
Comment (elicoidal) [inline crm_batch_leads/README.rst]: CRM batch process wizard

**Comment by noahzaozao** [comment]
LGTM @elicoidal @seb-elico @victormartinelicocorp

**Comment by noahzaozao** [comment]
https://github.com/Elico-Corp/elico_odoo/pull/5

**Pull request merged**